### DISCUSSION TITLE
MenuBar styling

### DISCUSSION THREAD

**Discussion by DasLixou** [qtile/qtile#3787] [Q&A]
Hello. I have visual studio code installed and my menu bar with items like file edit etc. are for vscode and other programs in the standard linux gray. is there a way to change them? has it something to do with the wm?

**Comment by m-col** [comment]
It's nothing to do with the WM, but many programs do let you configure the content of those menus. You will have to look at the docs for each program, they might mention a theme/colour scheme.
